reset default Prodigal and update usage.md (#210)
nf-core/funcscan: docs/usage.md

> ⚠️ If a sample does not reach this contig length threshold, you will recieve a warning in your console and `.nextflow.log` file, and no result files will exist for this sample in your results directory.

> We also noticed that when annotation is run with Prokka, the `.gbk` file passed to AntiSMASH may produce the error *'translation longer than location allows'* and end the pipeline run.

Reviewer: ```suggestion
Sometimes, when complex samples are run with the `--annotation_tool` set to Prokka, the `.gbk` file passed to AntiSMASH may produce the error *'translation longer than location allows'* and end the pipeline run.
```

Reviewer 2: I don't like using the term 'complex samples', I would rather say large assemblies or very long contigs or something like that

Author: The text has been modified already, so that this comment is outdated and no mention of "large, complex, long, samples, assemblies, contigs" is made any more, but the link to the respective discussion on the Prokka repo is given.

> We also noticed that when annotation is run with Prokka, the `.gbk` file passed to AntiSMASH may produce the error _'translation longer than location allows'_ and end the pipeline run.

Author: ```suggestion
When the annotation is run with Prokka, the resulting .gbk file passed to antiSMASH may produce the error 'translation longer than location allows' and end the pipeline run. This Prokka bug was recognised before (see https://github.com/antismash/antismash/discussions/450) and is not likely to be fixed soon.
```

> > ⚠️ If AntiSMASH is run for BGC detection it is recommended NOT to run PROKKA for annotation but instead leave the default annotation tool Prodigal or use BAKTA.

Reviewer: ```suggestion
> ⚠️ If AntiSMASH is run for BGC detection, we recommend NOT to run PROKKA for annotation but instead leave the default annotation tool Prodigal or use BAKTA.
```

> > ⚠️ If AntiSMASH is run for BGC detection it is recommended NOT to run PROKKA for annotation but instead leave the default annotation tool Prodigal or use BAKTA.

Reviewer 3: When the annotation is run with Prokka, its `.gbk` file of a sample passed to antiSMASH may produce the error _'translation longer than location allows'_ and end the pipeline run. This Prokka bug was recognized before (see [here](https://github.com/antismash/antismash/discussions/450)) and is not likely to be fixed soon.

> ⚠️ If antiSMASH is run for BGC detection it is recommended to not run Prokka for annotation but instead leave the default annotation tool Prodigal or switch to Bakta.
```

Reviewer 2: Include caveat that bakta is bacteria only

> When the annotation is run with Prokka, the resulting .gbk file passed to antiSMASH may produce the error 'translation longer than location allows' and end the pipeline run. This Prokka bug has been reported before (see https://github.com/antismash/antismash/discussions/450) and is not likely to be fixed soon.
> 
> > ⚠️ If AntiSMASH is run for BGC detection, we recommend to NOT run PROKKA for annotation but instead leave the default annotation tool Prodigal or switch to BAKTA.

Reviewer 3: ```suggestion
When the annotation is run with Prokka, the resulting `.gbk` file passed to antiSMASH may produce the error `translation longer than location allows` and end the pipeline run. This Prokka bug has been reported before (see [discussion on GitHub](https://github.com/antismash/antismash/discussions/450)) and is not likely to be fixed soon.

> ⚠️ If antiSMASH is run for BGC detection, we recommend to **not** run Prokka for annotation but instead leave the default annotation tool Prodigal or switch to Bakta.
```
I would keep the uppercase/lowercase of tool names consistent. I always refer to the metro map which has the correct writing.
